jerry-core/vm/vm.c

```diff
@@ -220,22 +218,16 @@ vm_run_global (ecma_value_t *error_value_p) /**< [out] error value */
                                    NULL,
                                    0);
 
-  if (ECMA_IS_VALUE_ERROR (ret_value))
-  {
-    *error_value_p = ret_value;
-    ret_code = JERRY_COMPLETION_CODE_UNHANDLED_EXCEPTION;
-  }
-  else
+  if (!ECMA_IS_VALUE_ERROR (ret_value))
   {
     ecma_free_value (ret_value);
-    *error_value_p = ecma_make_simple_value (ECMA_SIMPLE_VALUE_UNDEFINED);
-    ret_code = JERRY_COMPLETION_CODE_OK;
+    ret_value = ecma_make_simple_value (ECMA_SIMPLE_VALUE_TRUE);
```

Comment:
What about `jerry_run`? Should it give back the result of `vm_run_global` instead of true?